tools/loris_db_upgrade.php

```diff
@@ -14,38 +14,82 @@
  * @link     https://www.github.com/aces/Loris/
  * @package core
  */
-require_once __DIR__ . "/../vendor/autoload.php";
 require_once __DIR__ . "/generic_includes.php";
  set_include_path(get_include_path().":../project/libraries:../php/libraries:");
-require_once "../php/libraries/Database.class.inc";
-require_once "../php/libraries/NDB_Config.class.inc";
 $path    = __DIR__ . "/../SQL/Release_patches";
 
 $files = array_diff(scandir($path), array('.', '..'));
 
-echo "This script will upgrade the LORIS database to the latest.\n";
-echo "User must be granted all necessary privileges (ALTER TABLE, INSERT, UPDATE, DELETE ...) on Loris database.\n";
+echo "This script will upgrade the LORIS database to the latest release version.\n";
 
 if (!file_exists('../project/config.xml')) {
     exit("config.xml file doesn't exist. Aborting update.");
 }
 
-$v = readline("\n Enter current version of Loris database (ex:21.0): ");
-  
-$DB = \Database::singleton();
+$v = $argv[1];
+if (floatval($v) < 14.0 || floatval($v) > 99.0 )
```

Comment:
This isn't really correct. `14.19` and `56.0000000000000000000001` are invalid.

I think it would be better to read this from the filenames themselves.

You can split the filename on the `_` character and use only those exact numbers as valid values.